dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -124,6 +128,23 @@ public DdlStatementExecution createDdlStatementExecution(String ddl)
         return new DdlStatementExecution(getConnection().createStatement(), ddl);
     }
 
+    @Override
+    public StatementExecution createStatementExecution(PreparedStatement statement, boolean clearParameters) {
+        return new StatementExecution(statement, clearParameters);
+    }
+
+    @Override
+    public StatementExecution createFunctionStatementExecution(PreparedStatement statement, boolean clearParameters) {
+        return new StatementExecution(statement, clearParameters);
+    }
+
+    protected TypeTransformerFactory dbfitToJDBCTransformers = new TypeTransformerFactory();
```

Comment:
What about synchronizing with the naming as in other places `dbfitToJDBCTransformers` -> *TransformerFactory?